packages/internal/bridge/sdk/rollup.config.js

```diff
@@ -15,7 +15,7 @@ export default {
   plugins: [
     json(),
     commonjs(),
-    nodeResolve(),
+    nodeResolve({ browser: true}), 
```

Comment:
@proletesseract we had to make the node Resolve for browser so this would work in Checkout.

The problem is the bridge sample app scripts which run in Node will break. You can get it to work by removing browser: true. 

Most likely we will need to just create a CommonJS build for the node scripts and an ESM build for Checkout

Reply:
Or something along those lines